src/main/java/com/teachmeskills/student_info/controllers/StudentController.java

```diff
@@ -0,0 +1,84 @@
+package com.teachmeskills.student_info.controllers;
+
+import com.teachmeskills.student_info.model.Student;
+import com.teachmeskills.student_info.service.StudentService;
+import com.teachmeskills.student_info.util.StudentOperations;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.sql.SQLException;
+
+@Controller
+public class StudentController {
+    StudentService studentService = new StudentService();
+
+    @GetMapping("/create")
+    public String viewSetStudentPage() {
+        return "create";
+    }
+
+    @PostMapping("/create")
+    public ModelAndView createStudent(@RequestParam int id, @RequestParam String name, @RequestParam String surname, @RequestParam Integer groupID, @RequestParam Double grade, Model model) {
```

Comment:
Why do you have a lot of params instead of Student object?

Reply:
Fixed.